gulpfile.js

```diff
@@ -8,11 +8,26 @@ var sourcemaps = require('gulp-sourcemaps');
 var webserver = require('gulp-webserver');
 var jshint = require('gulp-jshint');
 var watch = require('gulp-watch');
+var karma = require('karma').server;
 
 var getBundleName = function () {
   return 'sww';
 };
 
+gulp.task('tests', function () {
+  karma.start({
+    configFile: __dirname + '/testing/karma.conf.js',
+    singleRun: true
+  });
+});
+
+gulp.task('sw-tests', function () {
```

Comment:
Do we need this task? We already have the tests one, this is getting a different karma config file, but I'm not sure if that 2nd config file is ok for this project. Will download and try.